--- zoom-1.1.5/src/tokenise.c.orig	2017-11-25 19:39:36.077243373 +0000
+++ zoom-1.1.5/src/tokenise.c	2017-11-25 19:39:40.196103553 +0000
@@ -154,7 +154,7 @@
 
 int cache = 1;
 
-inline ZUWord lookup_word(unsigned int*  word,
+static inline ZUWord lookup_word(unsigned int*  word,
 			  int            wordlen,
 			  ZUWord         dct)
 {
--- zoom-1.1.5/src/interp.c.orig	2024-02-21 23:52:06.046908309 +0000
+++ zoom-1.1.5/src/interp.c	2024-02-21 23:52:14.391637712 +0000
@@ -232,7 +232,7 @@
   return newframe;
 }
 
-inline void store(ZStack* stack, int var, ZWord value)
+static inline void store(ZStack* stack, int var, ZWord value)
 {
 #ifdef DEBUG
   printf_debug("Storing %i in Variable #%x\n", value, var);
